cmd/validate/image_test.go

```diff
@@ -261,6 +261,7 @@ func Test_ValidateImageCommand(t *testing.T) {
 		  {
 			"name": "Unnamed",
 			"containerImage": "registry/image:tag",
+			"source": {},
```

Comment:
Let's not leak this out to the user's output unless it's actually something we want.

Reply:
Wait, this includes the git source information which is indeed useful. It's empty in our tests because we don't provide any data to it. Presumably, this will be filled in if the input contains it. +1 to keep it as is.

We may also want to make sure this information makes its way to the input for the policy rules. (As another story, of course).